dashboard/src/App.vue

```diff
@@ -43,15 +43,21 @@ export default {
 </script>
 
 <style lang="scss">
+html, body {
+	//Generic css resets
+	box-sizing: border-box;
+	margin: 0;
+	padding: 0;
+	outline: 0;
+	overflow: hidden !important;
```

Comment:
No clue where overflow is being overwritten. Those two tags should never be scrollable.